deployments/container/Dockerfile.ubi8

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG GOLANG_VERSION=1.15.8
+ARG GOLANG_VERSION=1.17.9
```

Comment:
A useful addition may be a make target that checks whether this version is the same as in `version.mk` to avoid issues like this in future. (also for the CUDA version).

(Note we need this here because we provide instructions in the README to build the docker images directly).

Reply:
That's beyond my knowledge of makefiles (I always hated them since college :) ).
I could try to figure out something but it'll probably be an ugly and suboptimal bash script. I'd rather leave that to experts.